migrations/001_create_tests.up.sql

```diff
@@ -0,0 +1 @@
+create table tests (id varchar(10))
```

Comment:
Will this be included in each project importing `github.com/moov-io/base/database`?

Reply:
No. It just to make this package test pass (check that it really may migrate DB).